roombasim/pittras/task/xyz_translation_task.py

```diff
@@ -23,7 +25,7 @@ def __init__(self, target):
 
         # PID controller contstants
```

Comment:
If these are constants, do we want to hard code them in like this or do we want to have a param file that the constants get loaded in from? I do not know how often these will change, but why not at least set up the structure now to have params for your tasks that load in from a centralized param file, sorta like `motion_planner.yaml` for HLM.

Reply:
yep, moved these to `roombasim.pittras.config` since they will be used in other tasks as well

Reply:
yep, moved to config file